### DISCUSSION TITLE
Cam APS & xdrip

### DISCUSSION THREAD

**Discussion by Ktg26** [NightscoutFoundation/xDrip#2779] [General]
![Screenshot_20230414-165649_Amazon Appstore](https://user-images.githubusercontent.com/130787647/232127829-7633d7ab-688f-4099-bf89-db2452fa849f.jpg)
Struggling  to set up xdrip with cam aps,   ,for a smartwatch 

Samsung s10+
Android12

![20230414_182707-COLLAGE](https://user-images.githubusercontent.com/130787647/232127644-0ce8e1c0-4b24-45e9-af48-db63741064e9.jpg)

**Comment by Navid200** [comment]
Please also include a screenshot of the classic status page, which shows which version of xDrip you are using.

**Comment by Ktg26** [comment]
![inbound5046538444299258447](https://user-images.githubusercontent.com/130787647/232128327-40ac2589-02d4-4c5f-8405-e36954900a88.jpg)

**Comment by jamorham** [comment]
Is the units value in xDrip set to mmol?

**Reply by Navid200** [reply]
Thanks for that.
I will add it to the guide now.

**Reply by Ktg26** [reply]
Working now!!!!!!!!! Thank you so much for your help!!!!!! 
Amazing
Xxxxxx
